### ISSUE TITLE
editors.zed: Generic word find-native-bundle does not define a method for the word class. Dispatching on object: linux

### ISSUE THREAD

**Issue by AndydeCleyre** [factor/factor#3068]
On Linux, with `EDITOR: editors.zed`, Factor `0.100`, invoking the editor errors with:

```
Generic word find-native-bundle does not define a method for the word class.
Dispatching on object: linux
```

**Comment by mrjbq7** [comment]
Only supports macOS at the moment. 

Would love a patch for Linux support!

**Comment by mrjbq7** [comment]
Fixed in c3eb1104944ada772ad1d1a98cc0de0323dba2da.

You'll have to use git master, for the moment, to get this!

https://github.com/factor/factor?tab=readme-ov-file#building-factor-from-source
